### DISCUSSION TITLE
i cant see the pop up window when i press this button

### DISCUSSION THREAD

**Discussion by konos93** [community/community#114520] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Issues

### MDN URL

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Not_defined

### What information was incorrect, unhelpful, or incomplete?

Uncaught ReferenceError: HTMLDialogElement is not defined
    tO dialog_helper.js:22
    connectedCallback dialog_helper.js:83
    50064 dialog_helper.js:116
    Webpack 9

### What did you expect to see?

i cant press the buttonson github
![2024-03-27 20_33_07-Pull requests - Created — Mozilla Firefox](https://github.com/mdn/content/assets/18540037/0e77b59c-5291-4833-b7ce-517ba3bb1e98)

### Do you have any supporting links, references, or citations?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Not_defined?utm_source=mozilla&utm_medium=firefox-console-errors&utm_campaign=default

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/javascript/reference/errors/not_defined`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Not_defined
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/javascript/reference/errors/not_defined/index.md
* Last commit: https://github.com/mdn/content/commit/6d606174faaedaa5dee7b7ebd87602cd51e5dd7e
* Document last modified: 2023-07-24T23:45:00.000Z

</details>

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by RochaDiego04** [comment]
The HTMLDialogElement is a web API that provides methods to manipulate `<dialog> `elements. 
It’s possible that the browser you’re using does not support this API.
